kessler/cdm.py

```diff
@@ -14,7 +14,7 @@
 import datetime
 import copy
 import pandas as pd
-
+from . import time_utils
```

Comment:
I guess we do not need this import anymore, right?

Reply:
Yes this should removed.